spec/beaker/host/unix/exec_spec.rb

```diff
@@ -142,7 +142,7 @@ def to_s
     end
 
     describe '#prepend_commands' do
-      it 'returns the pc parameter unchanged for non-cisco platforms' do
+      it 'returns the pc parameter unchanged for all platforms' do
         allow(instance).to receive(:[]).with(:platform).and_return('notcisco')
```

Comment:
Can change this to something generic? Though this is also invalid, it's more reasonable IMHO.
```suggestion
        allow(instance).to receive(:[]).with(:platform).and_return('unix')
```